main.py

```diff
@@ -256,9 +262,21 @@ def start(self):
                 self.axes[i].clear()
                 self.axes[i].set(title = self.graph_title[i], )
                 if index < 50:
-                    self.axes[i].plot(self.numpy[i][0:index])
+                    self.axes[i].plot(self.numpy[i][0:50], 'b')
                 else:
-                    self.axes[i].plot(self.numpy[i][index-50:index])
+                    self.axes[i].plot(self.numpy[i][index-50:index], 'g-')
+                # I
+                if i == 1 and self.found_num < 50:
+                     self.axes[i].annotate(
+                     'max height: %d' % self.found_num,
+                     xy=(self.found_num, self.numpy[i][self.found_num]),
+                     xytext=(self.found_num-10, self.numpy[i][self.found_num]+3),
+                     arrowprops=dict(facecolor='black', shrink=0.05))
+                elif i == 0:
+                    self.axes[i].axhline(y=self.stillness, c='r', ls='--')
+
+                if index < len(self.numpy[i]):
+                    self.axes[i].plot([index], self.numpy[i][index], 'r^')
```

Comment:
It would be much easier to understand if you could upload the captions together to see how they have changed.

Reply:
I add Comment about it, thank you